staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/types.go

```diff
@@ -58,6 +58,9 @@ type ExecCredentialStatus struct {
 	ExpirationTimestamp *metav1.Time `json:"expirationTimestamp,omitempty"`
 	// Token is a bearer token used by the client for request authentication.
 	Token string `json:"token,omitempty"`
+	// PEM-encoded client TLS certificate and key.
+	ClientCertificateData string `json:"clientCertificateData,omitempty"`
+	ClientKeyData         string `json:"clientKeyData,omitempty"`
```

Comment:
Is this a private key? Can you explain in the documentation why it is safe to transmit a private key?

Reply:
Yes. Added a comment to wrapper struct.
This is not a message that will get transmitted over the network. It's for exec plugins (same host). The whole purpose of it is to generate key/cert and give it to client (kubelet).